packages/lisk-api-client/package.json

```diff
@@ -19,24 +19,48 @@
 		"node": ">=6.3 <=9.5",
 		"npm": ">=3 <=5"
 	},
-	"main": "dist/index.js",
+	"browser": "dist-browser/index.min.js",
+	"main": "dist-node/index.js",
 	"scripts": {
+		"transpile": "babel src -d dist-node",
+		"transpile:browsertest":
+			"babel src -d ./browsertest/src && BABEL_ENV=browsertest babel test -d ./browsertest/test",
+		"browserify":
+			"browserify ./dist-node/index.js -o ./dist-browser/index.js -s lisk --node",
+		"browserify:browsertest":
+			"browserify ./browsertest/test/*.js ./browsertest/test/**/*.js -o ./browsertest/browsertest.js -s lisk",
+		"uglify":
+			"uglifyjs -nm -o ./dist-browser/index.min.js ./dist-browser/index.js",
+		"uglify:browsertest":
+			"uglifyjs -o ./browsertest/browsertest.min.js ./browsertest/browsertest.js",
 		"format": "prettier --write \"*.{js,json}\" \"{src,test}/**/*.{js,json}\"",
 		"lint": "eslint .",
 		"lint:fix": "npm run lint -- --fix",
 		"test": "NODE_ENV=test nyc mocha test",
 		"test:watch": "npm test -- --watch",
 		"test:watch:min": "npm run test:watch -- --reporter=min",
+		"test:node":
+			"npm run build:check && BABEL_ENV=buildcheck mocha test --compilers js:babel-register --recursive",
```

Comment:
This doesn't make sense anymore. We should reinstate the buildcheck `BABEL_ENV`: https://github.com/LiskHQ/lisk-elements/blob/1.0.0/.babelrc#L17-L26